docs/cookbook.md

```diff
@@ -18,7 +18,11 @@ Here's how they would look using Temporal.
 How to get the current date and time in the local time zone?
 
 ```javascript
-{{cookbook/getCurrentDate.mjs}}
```

Comment:
These have to be reverted, it will break the cookbook. Probably prettier should not touch this file, because of this include syntax.

Reply:
Crap. I'll revert. Sorry for not reviewing the prettier changes more closely-- they were done by a global search-n-replace which invoked prettier when saving changes.